### PR TITLE
Make progress bars sexier

### PR DESCRIPTION
**Pull request by rehandalal**
r?

### CODE REVIEW

File: postatus/templates/l10n_summary.html [hunk 1]
```diff
@@ -45,10 +44,8 @@
                     {{ item.untranslated }}
                   </td>
                   <td>
-                    <span class="perc perc-{% if bars == 20 %}good{% elif bars > 15 %}ok{% else %}bad{% endif %}">
-                      {% for chunk in range(bars) %}
-                        <span class="perc-notch">=</span>
-                      {% endfor %}
+                    <span class="perc perc-{% if item.percent == 100 %}good{% elif item.percent > 75 %}ok{% else %}bad{% endif %}">
+                      <span style="width: {{ item.percent }}%"></span>
```
Comment (willkg) [inline postatus/templates/l10n_summary.html]: This only covers the highlighted locales. You also need to update the locales section below.

**Comment by willkg** [comment]
One minor issue, but this looks super! Thank you!

File: postatus/templates/l10n_summary.html (hunk 1 above)
Comment (rehandalal) [inline postatus/templates/l10n_summary.html]: yeah i saw that and fixed already

**Comment by willkg** [comment]
Yay! THANK YOU!

**Pull request merged**